message_definitions/v1.0/common.xml

```diff
@@ -2884,6 +2884,79 @@
         <description>Use precision landing, searching for beacon if not found when land command accepted (land normally if beacon cannot be found).</description>
       </entry>
     </enum>
+    <enum name="OPENDRONEID_ID_TYPE">
```

Comment:
There's a distinct pattern in mavlink where the complete enumeration name is used as a prefix for the entries.

Reply:
Fixed